src/styles/variables.scss

```diff
@@ -25,3 +25,17 @@ $break-max: 1110px;
 $break-min: 768px;
 $above-max-break: "only screen and (min-width: #{$break-max})";
 $below-min-break: "only screen and (max-width: #{$break-min})";
+
```

Comment:
was there a particular reason you are exporting here instead of keeping the js file? I feel like that one is more clear, and importing `colors` is more descriptive than `variables`

Reply:
I extracted variables.js into variables.scss so that we aren't using to different breakpoints for our css and js. I named it variables, because there is value that is not a color. I tried nesting these into sub-properties and failed. Hence, they are all under variables.